.pre-commit-config.yaml

```diff
@@ -14,22 +14,25 @@ repos:
       - id: trailing-whitespace
       - id: check-added-large-files
   - repo: https://github.com/astral-sh/uv-pre-commit
-    rev: 0.7.18
+    rev: ed07c6e8889cf83a7245866d1dd007d1f93bf55e  # frozen: 0.9.13
     hooks:
       - id: uv-lock
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.11.11
+    rev: 36243b70e5ce219623c3503f5afba0f8c96fda55  # frozen: v0.14.7
     hooks:
       - id: ruff-check
         args: [ --fix ]
       - id: ruff-format
+  # mdformat plugins are not all compatible with mdformat 1.0.0 yet.
   - repo: https://github.com/hukkin/mdformat
-    rev: 0.7.22
+    rev: 83dc2ce5ca115d3390b9b5561d2898366faf0742  # frozen: 0.7.2
```

Comment:
typo in your comment; this hash is [0.7.22](https://github.com/hukkin/mdformat/commit/83dc2ce5ca115d3390b9b5561d2898366faf0742), not the much older [0.7.2](https://github.com/hukkin/mdformat/commit/8688cc20a9a4fb775dbb0e6b559fd47144147515) :)